src/theory/fp/theory_fp_type_rules.h

```diff
@@ -464,8 +464,38 @@ class FloatingPointToRealTypeRule {
   }
 };
 
-} /* CVC4::theory::fp namespace */
-} /* CVC4::theory namespace */
-} /* CVC4 namespace */
+
+class CardinalityComputer {
```

Comment:
The current draft of the code style guides for CVC4 [discourages](https://github.com/CVC4/CVC4/wiki/Code-Style-Guidelines#nonmember-static-member-and-global-functions) using classes with only static methods.